Cosmostation/Model/SupportPool.swift

```diff
@@ -9,11 +9,13 @@
 import Foundation
 
 public struct SupportPool {
+    var type: String!
     var id: String!
     var adenom: String!
     var bdenom: String!
     
     init(_ dictionary: NSDictionary?) {
+        self.type = dictionary?["type"] as? String ?? ""
```

Comment:
if always using a fallback to empty string you must remove `var type: String!` with `let type: String`, usage of `!` should be avoided if is not necesary, it can lead to crashes, not in this case as you are using a fallback. The point is to avoid coding stuff not needed and in this case `!` in the `var type: String` it isnt